src/Nethermind/Nethermind.Era1/AccumulatorCalculator.cs

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cortex.SimpleSerialize;
+using Nethermind.Core.Crypto;
+using Nethermind.Int256;
+
+namespace Nethermind.Era1;
+//See https://github.com/ethereum/portal-network-specs/blob/master/history-network.md#algorithms
+internal class AccumulatorCalculator
+{
+    List<SszComposite> _roots;
```

Comment:
Maybe use `ArrayPoolList`, make it `IDisposable` and dispose it when not needed. Would save us on allocating this list every time.

Reply:
Done